extensions/ql-vscode/src/common/commands.ts

```diff
@@ -198,6 +199,13 @@ export type QueryHistoryCommands = {
   "codeQL.exportSelectedVariantAnalysisResults": () => Promise<void>;
 };
 
+// Commands user for the language selector panel
+export type LanguageSelectionCommands = {
```

Comment:
This type should also be included in `AllExtensionCommands`, I'm not sure why adding the `getCommands` call in `extension.ts` didn't give you an error that it isn't included.